report.c

```diff
@@ -238,6 +238,8 @@ void report_grbl_settings() {
   printPgmString(PSTR(" (magazine gap enabled, bool)"));
   printPgmString(PSTR("\r\n$42=")); print_uint8_base10(settings.use_load_cell);
   printPgmString(PSTR(" (use load cell, bool)"));
+  printPgmString(PSTR("\r\n$43=1"));
```

Comment:
I feel like this should be a bigger number (I'm sure this won't be the final be-all-end-all settings end)